tests/estimation/test_get_estimation.py

```diff
@@ -249,3 +249,13 @@ def test_total_is_direct_plus_indirect(effects_chap):
         assert effects_chap[0] == pytest.approx(effects_chap[1] + effects_chap[4])
     if not np.isnan(effects_chap[2]):
         assert effects_chap[0] == pytest.approx(effects_chap[2] + effects_chap[3])
+
+
+@pytest.mark.xfail
+def test_robustness_to_ravel_format(data, estimator, config, effects_chap):
+    if "forest" in estimator:
```

Comment:
Is there a true reason for that ?If not, this is an issue (to be opened)

Reply:
I would say the forest behavior is similar to other estimators. Makes sense as all scikit-learn models have the same behavior (I think) withing a single implemented estimator, so we slightly reduce the number of run tests. Would you rather still test all configurations?

Reply:
They have to be tested once (but no more than once).

Reply:
> Is there a true reason for that ?If not, this is an issue (to be opened)

Yes, forest estimators return a random result, so testing `assert get_estimation(data[0], data[1], data[2], data[3], estimator, config)[0:5] == pytest.approx(effects_chap, nan_ok=True)` would always fail